README.md

```diff
@@ -17,237 +17,106 @@ A distributed implementation of Redis compatible layer based on [TiKV](https://g
 
 Thanks [TiKV](https://github.com/tikv/tikv/) for supporting the core features. The project is developed and open sourced by the Beijing Infrastructure Team at [Meitu](https://www.meitu.com/) and has been donated to [DistributedIO](https://github.com/distributedio) org.
 
-## Roadmap
+## Architecture
 
-[Roadmap](docs/roadmap.md)
+___Arch about titan___
+
+![titan](docs/titan.png)
+
+___Arch about TiKV___
+
+![TiKV softwares](https://github.com/tikv/tikv/blob/master/images/tikv_stack.png)
+
+## Quick start
+
+Can't wait to experiment Titan? Just follow 2 steps:
+
+1. curl -s -O https://raw.githubusercontent.com/distributedio/titan/master/docker-compose.yml
+2. docker-compose up
 
-## Can't wait to experiment Titan?
+Then connect to titan use redis-cli
 
+> redis-cli -p 7369
+
+___Enjoy!___
+
+## Installation
+
+### SetUp TiKV cluster
+Titan works with 2 tidb components:
+* TiKV
+* Pd
+
+To setup TiKV and PD, please follow official [instruction](https://pingcap.com/docs-cn/dev/how-to/deploy/orchestrated/ansible/)
+
+### Run Titan
+
+* Build the binary
+
+```
+go get github.com/distributedio/titan
+cd $GOPATH/src/github.com/distributedio/titan
+make 
 ```
-curl -s -O https://raw.githubusercontent.com/distributedio/titan/master/docker-compose.yml
-docker-compose up
 
-# Then connect to titan use redis-cli
-redis-cli -p 7369
+* Edit the configration file
 
-# Enjoy!
 ```
+pd-addrs="tikv://your-pd-addrs:port"
+```
+
+* Run titan
 
-## Installing
+```
+./titan
+```
 
-[Deploy Titan](docs/ops/deploy.md)
+For more details about [Deploy Titan](docs/ops/deploy.md), click here.
 
 ## Benchmarks
 
-[Titan Benchmarks](docs/benchmark/benchmark.md)
+Basic benchmarking result.
+
+### Get
+
+![Get command benchmark](docs/benchmark/get-benchmark.png)
+
+### Set
+
+![Set command benchmark](docs/benchmark/set-benchmark.png)
+
+For more info, please vist here [Titan Benchmarks](docs/benchmark/benchmark.md)
+
+## Commands not supported yet
```

Comment:
This title is not proper anymore.
Maybe `Command support status`? or something else.

Reply:
fixed